ChangeLog

```diff
@@ -1,6 +1,8 @@
 CHANGES
 =======
 
+* Drop support for marshmallow 2
```

Comment:
Please remove changes in this file. ChangeLoge will be generated automatically

Reply:
^^^ please revert. The bump version script automatically generates the change log file